sig-docs/annual-report-2021.md

```diff
@@ -4,79 +4,72 @@
 
-   -
-   -
-   -
+
+   - SIG Docs has a [dashboard](https://datastudio.google.com/u/0/reporting/fede2672-b2fd-402a-91d2-7473bdb10f04/page/567IC) available with site analytics. Some highlights include: 
+       - 2021 Pages views: 111,565,437
+       - Top pages for 2021 (excluding the home page): 
+           - https://kubernetes.io/docs/reference/kubectl/cheatsheet/
+           - https://kubernetes.io/docs/concepts/services-networking/service/
+           - https://kubernetes.io/docs/concepts/services-networking/ingress/
+   - PR velocity and open PR age is tracked in [Devstat](https://k8s.devstats.cncf.io/d/25/open-pr-age-by-repository-group?orgId=1&var-period=q&var-repogroup_name=SIG%20Docs&var-kind_name=All). We aim to have < 100 open PRs for the English localization, and will take steps as needed if we see the figure climbing much above that. For example, based on 12/31/2020 data, we have: Average number of opened PRs 368 with the median opened PRs age 5 days 21 hours 26 minutes 42 seconds
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - It's updated to the best of our knowledge.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - We have a [well-documented guide](https://kubernetes.io/docs/contribute/#:~:text=Anyone%20can%20open%20an%20issue,the%20CNCF%20Contributor%20License%20Agreement.) that details how folks can get started and the various ways in which they can scale the contributor ladder.
+   - We actively assign good-first-issue labels to issues that we think early-lifecycle contributors could begin with. During KubeCons we also do introduction presentations and mentoring sessions to try and funnel contributors in to our SIG.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - [98 companies](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=SIG%20Docs&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&from=1609455600000&to=1639350000000)
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - We're in the process of formalizing the localization subproject. We aim for it to, potentially, provide more avenues for contribution. More details around leadership for the localization supbproject can be found in [this message](https://groups.google.com/a/kubernetes.io/g/dev/c/SP6weMvx3wg/m/l8LAL-OFCQAJ) sent to dev@kubernetes.io 
```

Comment:
is there a statement we can make in this section about the importance of localization and why end users should care?

Reply:
Yes, I will draft something.